docs/recipes/travis-build-stages.md

```diff
@@ -1 +1,63 @@
-# Using semantic-release with [Travis CI build stages](https://docs.travis-ci.com/user/build-stages)
+# Using semantic-release with [Travis build stages](https://docs.travis-ci.com/user/build-stages/)
+
+![Travis Build Stages screenshot](https://cloud.githubusercontent.com/assets/3729517/25229553/0868909c-25d1-11e7-9263-b076fdef9288.gif)
+
+Build stages are a great way to run semantic release compared to running it in `after_success`:
+
+* The release stage can use Node 8, while the other stages can use other versions, even use a completely different language.
+* It allows you to easily release a project that is not even written in JavaScript, without impacting the job configuration of your test jobs in any way.
+* Instead of having to figure out a "build leader" to do the release and poll the Travis API to wait until all jobs passed, Travis can handle only running the release step after the other jobs passed.
+* If the release failed, you can see at a glance that only the release stage failed while your test stage succeded.
+* Travis can take care of not running the whole release stage at all on non-master branches and PRs.
+
+### Annotated example `.travis.yml`
+
```

Comment:
Can you add a section about `package.json` configuration mentioning it's required only for local installs similarly to [travis.md](https://github.com/semantic-release/semantic-release/blob/caribou/docs/recipes/travis.md#packagejson-configuration-for-single-node-job):

```md
### `package.json` configuration

A `package.json` is required only for [local](../usage/installation.md#local-installation) **semantic-release** installation.

{
  "devDependencies": {
    "semantic-release": "^11.0.0"
  },
  "scripts": {
    "semantic-release": "semantic-release"
  }
}
```